roles/edge-image-builder/templates/eib-config-mgmt.yaml.j2

```diff
@@ -0,0 +1,26 @@
+apiVersion: 1.0
+image:
+  imageType: raw
+  arch: x86_64
+  baseImage: {{ eib_source_image_name }}
+  outputImageName: {{ eib_mgmt_output_image_name }}
+operatingSystem:
+  rawConfiguration:
+    diskSize: {{ eib_image_size }}
```

Comment:
do you need it? with the growfs is not enough?

Reply:
@alknopfler this is the management cluster config, where this is needed so EIB resizes the disk at build time (otherwise we run out of space adding the RKE2 dependencies)